### PR TITLE
Adding cleanup jobs in cleanup script.

### PR DESCRIPTION
**Pull request by rajdharwadkar**

### CODE REVIEW

File: script_library/clean-airship.sh [hunk 1]
```diff
@@ -79,7 +79,7 @@ if [[ ${clean_action} == *"clean_openstack"* ]]; then
     # DO NOT USE clusterrolebinding, else you will delete all rolebindings, even the suse: and system: ones,
     # even when scoped in the namespace.
     kubectl get -n openstack rolebinding.rbac.authorization.k8s.io -o name | xargs kubectl -n openstack delete
-
+    kubectl get jobs -n openstack | awk 'NR >= 1 {print $1 }'| xargs kubectl delete job -n openstack
```
Comment (evrardjp) [inline script_library/clean-airship.sh]: why not -o name instead of awk?  Save electrons!

File: script_library/clean-airship.sh [hunk 2]
```diff
@@ -88,6 +88,7 @@ if [[ ${clean_action} == *"clean_rest"* ]]; then
 fi
 
 if [[ ${clean_action} == *"clean_ucp"* ]]; then
+    kubectl get jobs -n ucp | awk 'NR >= 1 {print $1 }'| xargs kubectl delete job -n ucp
```
Comment (evrardjp) [inline script_library/clean-airship.sh]: why not -o name instead of awk?  Save electrons!

**Comment by evrardjp** [comment]
I like this.

File: script_library/clean-airship.sh (hunk 1 above)
Comment (kantarun) [reply script_library/clean-airship.sh]: haha ..electrons crusader ..liked it.

I guess with -o name, delete command will change as well as I see following output 
``
linux-bybo:/opt # kubectl get jobs -n ucp -o name
job.batch/keystone-bootstrap
job.batch/keystone-credential-setup
``
kubectl get jobs -n openstack -o name | xargs kubectl delete -n openstack

**Pull request merged**